### PR TITLE
:sparkles: Add positions pages

### PR DESCRIPTION
**Pull request by Macludde**
Also added:
- Links from profile page to positions page
- Re-did logic for mandate history. Now mandates from summer-summer are displayed correctly.
- Add possibility to edit positions (and emails)
- Only show "primary" email in most places, "all" emails are only shown on positions page

### CODE REVIEW

File: frontend/api/members.graphql [hunk 1]
```diff
@@ -12,6 +12,9 @@ query MeHeader {
         id
         name
         nameEn
+        committee {
+          name
+        }
```
Comment (danieladugyan) [inline frontend/api/members.graphql:17]: Do we need to fetch this in the Me-query?

File: frontend/components/Mandates/MandateList.tsx [hunk 2]
```diff
@@ -65,21 +67,34 @@ export default function MandateList({ year }: { year: number }) {
...
         </TableHead>
         <TableBody>
-          {positions.map((p, i) =>
-            (mandatesByPosition.has(p) ? (
+          {positions.map((p, i) => {
+            const posId = mandateList.find((m) =>
+              (isEnglish && m.position.nameEn
+                ? m.position.nameEn
+                : m.position.name) === p)
+              ?.position?.id;
```
Comment (danieladugyan) [inline frontend/components/Mandates/MandateList.tsx:75]: Maybe extract to method with a name that clarifies what this does?

File: frontend/public/locales/sv/position.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "otherEmails": "Dessa e-poster går också till denna position",
+  "primaryEmail": "Primära e-post"
```
Comment (danieladugyan) [inline frontend/public/locales/sv/position.json:3]:
```suggestion
  "primaryEmail": "Primär e-post"
```
(primära är plural)

File: frontend/public/locales/en/position.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "otherEmails": "These emailadresser also go to this position",
```
Comment (danieladugyan) [inline frontend/public/locales/en/position.json:2]:
```suggestion
  "otherEmails": "These email addresses also go to this position",
```

**Comment by danieladugyan** [comment]
> Also added:
> 
> * Links from profile page to positions page
> * Re-did logic for mandate history. Now mandates from summer-summer are displayed correctly.
> * Add possibility to edit positions (and emails)
> * Only show "primary" email in most places, "all" emails are only shown on positions page

Regarding the last point; is this done by fetching position email from the positions table instead of from email_aliases?

**Comment by Macludde** [comment]
> > Also added:
> > 
> > * Links from profile page to positions page
> > * Re-did logic for mandate history. Now mandates from summer-summer are displayed correctly.
> > * Add possibility to edit positions (and emails)
> > * Only show "primary" email in most places, "all" emails are only shown on positions page
> 
> Regarding the last point; is this done by fetching position email from the positions table instead of from email_aliases?

Yes exactly. I looked into it and the "email" column in positions makes a lot of sense. This field can also be edited on each position to on a per-basis level "choose" the primary email.

File: frontend/api/members.graphql (hunk 1 above)
Comment (Macludde) [reply frontend/api/members.graphql:17]: No, you are correct. Removing.

File: frontend/components/Mandates/MandateList.tsx [hunk 5]
```diff
@@ -28,6 +29,13 @@ type PartialMandate = {
   member?: Partial<Member>;
 };
 
+const getPositionIdFromName = (mandateList: GetMandatesByPeriodQuery['mandatePagination']['mandates'], positionName: string, isEnglish: boolean) => {
+  const translatedPositonName = (m: GetMandatesByPeriodQuery['mandatePagination']['mandates'][number]) =>
+    (isEnglish && m.position.nameEn ? m.position.nameEn : m.position.name);
+  const posId = mandateList.find((m) => translatedPositonName(m) === positionName)?.position?.id;
```
Comment (danieladugyan) [inline frontend/components/Mandates/MandateList.tsx:35]:
```suggestion
  const translatedPositionName = (m: GetMandatesByPeriodQuery['mandatePagination']['mandates'][number]) =>
    (isEnglish && m.position.nameEn ? m.position.nameEn : m.position.name);
  const posId = mandateList.find((m) => translatedPositionName(m) === positionName)?.position?.id;
```

File: frontend/public/locales/en/position.json [hunk 6]
```diff
@@ -1,4 +1,4 @@
 {
-  "otherEmails": "These emailadresser also go to this position",
+  "otherEmails": "These email adresser also go to this position",
```
Comment (danieladugyan) [inline frontend/public/locales/en/position.json:2]:
```suggestion
  "otherEmails": "These email adresses also go to this position",
```

**Review by danieladugyan — APPROVED** [review]
Some mostly insignificant typos left, but other than that it looks good!

**Pull request merged**